validator/src/main/resources/validations/otlp-prometheus-validation.yml

```diff
@@ -0,0 +1,7 @@
+-
+  validationType: "prom-static-metric"
+  httpPath: "/outgoing-http-call"
+  httpMethod: "get"
+  callingType: "http"
+  shouldValidateMetricValue: false
```

Comment:
Values can be random since we are testing a latency metric. It is not a static value.